crates/sui-rosetta/src/types.rs

```diff
@@ -591,7 +593,6 @@ impl IntoResponse for ConstructionMetadataResponse {
 pub enum TransactionMetadata {
     PaySui(Vec<ObjectRef>),
     Delegation {
-        sui_framework: ObjectRef,
```

Comment:
@patrickkuo -- wanted to check that I'm doing something OK here for Rosetta -- now that the we don't need ObjectRefs for packages, I stripped away this field, because it will always be the same value, if it were an ID.

Reply:
Ya, the only reason we have this is because of object ref, changing it to object id simplify things a lot. 🎉